scripts/redux/constants.js

```diff
@@ -27,3 +28,11 @@ const FETCH_SPEAKERS_LIST = 'FETCH_SPEAKERS_LIST';
 const FETCH_GALLERY = 'FETCH_GALLERY';
 // team
 const FETCH_TEAM = 'FETCH_TEAM';
+// sign in
+const SIGN_IN = 'SIGN_IN';
+// sign out
```

Comment:
the idea of this comments is to divide by re reduces, so you can remove this and next comment